CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
 ## Unreleased changes
+- Stronger typing for various places in the API. 
+- Renamed GTUAmount to CCDAmount
+- New CredentialRegistrationId type.
+- BlockSummary API now reflects the underlying types better.
+- getAccountInfo can now be queried either via `AccountAddress` or `AccountIndex` via `AccountRequest`.
```

Comment:
The node also supports querying via credential registration id. Perhaps that should be part of it here as well.